app/src/services/version.js

```diff
@@ -268,27 +269,31 @@ const service = {
    * @param {string} options.id COMS uuid of a version
    * @param {string} options.objectId COMS uuid of an object
    * @param {boolean} options.isLatest isLatest value to set in db
-   * @param {string} [userId=SYSTEM_USER] The uuid of a user
    * @param {object} [etrx=undefined] An optional Objection Transaction object
    * @returns {object} Version model of updated version in db
    */
-  updateIsLatest: async ({ id, objectId, isLatest }, userId = undefined, etrx = undefined) => {
+  updateIsLatest: async ({ id, objectId, isLatest }, etrx = undefined) => {
```

Comment:
I'm not sure if we should be dropping userId attribution for this. Yes, this is technically a maintenance function for keeping things consistent, but the consistency trigger will still generally be invoked by some agent for some reason or another. I think we chatted about this previously as we were worried about update misattribution, so it's fine to leave it not updated I suppose, but removing it from the function signature I'm not certain of yet.

Reply:
I'm 100% sure either but i think this is best.
a few points to try and clarify:

- This function is typically called when a new version is created and you set it as latest.
the new version will have `createdBy: <user>`. 

- If the object is in an un-versioned bucket (there will only be one version), after an updateObject call, the single version does show `updatedBy: <user>`. 

- it is a maintainence process, I dont believe we need to update the updatedBy column again when we're just setting as latest in this separate process.

- we could just set `updatedBy: <user>` for the version that will become latest. but we definitely dont want to update that column for all the other versions we are setting as not latest.

- the object record (not version) also has createdBy and updatedBy attributes which are probably more useful.

Reply:
Let's remove the argument for now, but add in a TODO comment considering re-adding it back in down the line so that we don't forget about this consideration in the future. 👍